res/controllers/Reloop-Mixage.scripts.js

```diff
@@ -0,0 +1,780 @@
+    "pfl": {"function": function(v, g, c) { Mixage.toggleLED(v, g, c); }},
+    "loop_enabled": {"function": function(v, g) { Mixage.toggleLoopLEDS(v, g); }},
+    "loop_in": {"function": function(v, g) { if (v === 1) { Mixage.toggleLED(ON, g, "loop"); } }},
+    "loop_out": {"function": function(v, g) { if (v === 1) { Mixage.toggleLED(ON, g, "reloop"); } }},
+    "sync_enabled": {"function": function(v, g, c) { Mixage.toggleLED(v, g, c); }},
+};
+
+// ----- Internal variables functions -----
+
+// Set or remove functions to call when the state of a mixxx control changes
+Mixage.connectControlsToFunctions = function(group, remove) {
+    for (const control in Mixage.connectionMap) {
+        if (remove !== undefined) {
+            Mixage.connectionMap[control][group].disconnect();
+        } else {
+            Mixage.connectionMap[control][group] = engine.makeConnection(group, control, Mixage.connectionMap[control].function);
+        }
+    }
+};
+
+Mixage.init = function(_id, _debugging) {
+
+    // all button LEDs off
+    for (let i = 0; i < 255; i++) {
+        midi.sendShortMsg(0x90, i, 0);
+    }
+
+    // find controls and make engine connections for each channel in Mixage.channels
+    // A predefined list with channels is used instead of a for loop to prevent engine connections to be overwritten
+    Mixage.channels.forEach(function(channel) {
+        const deck = script.deckFromGroup(channel);
+        Mixage.connectControlsToFunctions(channel);
+        const effectUnit = `[EffectRack1_EffectUnit${deck}]`;
+
+        // set soft takeovers for effectslot amount
+        for (let effectSlot = 1; effectSlot <= Mixage.numEffectSlots; effectSlot++) {
+            const groupString = `[EffectRack1_EffectUnit${deck}_Effect${effectSlot}]`;
+            engine.softTakeover(groupString, "meta", true);
+        }
+
+        for (let effectUnit = 1; effectUnit <= Mixage.numEffectUnits; effectUnit++) {
+            // make connections for the fx on LEDs
+            const fxControl = `group_${channel}_enable`;
+            const fxGroup = `[EffectRack1_EffectUnit${effectUnit}]`;
+            Mixage.fxOnConnection.push(engine.makeConnection(fxGroup, fxControl, function() { Mixage.toggleFxLED(channel); }));
+
+            // set soft takeovers for effectunit meta
+            engine.softTakeover(fxGroup, "super1", true);
+            engine.setValue(fxGroup, "show_focus", 1);
+        }
+
+        // set soft takeover for filter effect
+        engine.softTakeover(`[QuickEffectRack1_${channel}]`, "super1", true);
+
+        // make connections for status LEDs
+        Mixage.vuMeterConnection.push(engine.makeConnection(channel, "vu_meter", function(val) { midi.sendShortMsg(0x90, Mixage.ledMap[channel].vu_meter, val * 7); }));
+        Mixage.trackLoadedConnection.push(engine.makeConnection(channel, "track_loaded", function() { if (Mixage.adjustLoop[channel]) { Mixage.stopLoopAdjust(channel); } }));
```

Comment:
IIUC `makeUnbufferedConnection` is preferred for VU connections
example: https://github.com/mixxxdj/mixxx/blob/77b466e745dc9cf8bdf57286426cdca4b12f485f/res/controllers/Traktor-Kontrol-Z1-scripts.js#L113
Doc: https://github.com/mixxxdj/mixxx/blob/77b466e745dc9cf8bdf57286426cdca4b12f485f/res/controllers/engine-api.d.ts#L118-L129

Reply:
Can not test this currently. On holiday without the controller. Can check this in a couple of days

Reply:
Using an unbuffered connection now